disnake/member.py

```diff
@@ -288,6 +288,7 @@ class Member(disnake.abc.Messageable, _UserTag):
         banner: Optional[Asset]
         accent_color: Optional[Colour]
         accent_colour: Optional[Colour]
+        avatar_decoration: Optional[Asset]
```

Comment:
Looks like `Member.avatar_decoration` is missing in the docs; this would then need updating as well:
https://github.com/DisnakeDev/disnake/blob/9de76a1b3a24859e166309c3d4a24c411737f400/docs/api.rst?plain=1#L5485

Reply:
I can see it in the docs 🤔 
![image](https://user-images.githubusercontent.com/67214928/209434910-02c6d7e1-61f6-4306-b07c-e9c8b906c8ee.png)

Do you mean this documentation?
![image](https://user-images.githubusercontent.com/67214928/209434930-674d1ee3-9fec-46e4-bdcb-6e098d67c8cf.png)

Reply:
> I can see it in the docs

Ah, nevermind, my bad. In that case, only `WidgetMember.avatar_decoration` should be hidden from the docs like above.

Reply:
👍

Reply:
49455f6d7ae75ddb62281d6675ea492c184c406c